### PR TITLE
allow listitems to live in menu (fix #7802)

### LINKED ISSUE #7802: content: Allow li items in menu element
### What page(s) need to be updated?

https://web.dev/listitem/

### Why is this update needed?

The `<menu>` element is a semantic alternative to `<ul>`, which means that `<li>` elements are allowed to live inside of it, according to the [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu) and [W3 spec](https://html.spec.whatwg.org/multipage/grouping-content.html#the-menu-element). Axe-core has also acknowledged and fixed this in version 4.4.0 (issues https://github.com/dequelabs/axe-core/issues/3411 and https://github.com/dequelabs/axe-core/issues/3284, PR https://github.com/dequelabs/axe-core/pull/3286). Lighthouse seems to have updated axe-core to at least 4.4.1 (commit https://github.com/GoogleChrome/lighthouse/commit/37a1ceb4bb365fc6df470d1cc172760ae03fffa5) in the upcoming version. Assuming this also means Lighthouse will allow `<li>` to live in `<menu>` elements, I think this should also be reflected in the web.dev content.

### What's the deadline?

The release of Lighthouse 9.6.0, can’t for the life of me figure out when that would be (I guess April? But as it’s late in April, maybe May?)

#### Is it a hard deadline?

No, would be nice though.

### PR DESCRIPTION
**Pull request by FMJansen**
List items are also allowed to exist within a menu element, which is now updated in axe-core.

Fixes #7802

Changes proposed in this pull request:

- add <menu> to the list of elements in which a listitem can exist

### CODE REVIEW

**Comment by FMJansen** [comment]
Concerning the devrel-review-bot: https://github.com/GoogleChrome/lighthouse/issues/13925

**Comment by FMJansen** [comment]
This change is still needed, I think by now Chrome has updated, too.

File: src/site/content/en/lighthouse-accessibility/listitem/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ web_lighthouse:
 
 Screen readers and other assistive technologies
 require list items (`<li>`) to be contained
-within parent `<ul>` or `<ol>` to be announced properly.
+within parent `<ul>`, `<ol>` or `<menu>` to be announced properly.
```
Comment (heyawhite) [inline src/site/content/en/lighthouse-accessibility/listitem/index.md:15]:
```suggestion
within parent `<ul>`, `<ol>`, or `<menu>` to be announced properly.
```

File: src/site/content/en/lighthouse-accessibility/listitem/index.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ assistive technologies can't set user expectations correctly.
 ## How this Lighthouse audit fails
 
 Lighthouse flags list items (`<li>`) that aren't contained
-in `<ul>` or `<ol>` parent elements:
+in `<ul>`, `<ol>` or `<menu>` parent elements:
```
Comment (heyawhite) [inline src/site/content/en/lighthouse-accessibility/listitem/index.md:25]:
```suggestion
in `<ul>`, `<ol>`, or `<menu>` parent elements:
```

File: src/site/content/en/lighthouse-accessibility/listitem/index.md [hunk 3]
```diff
@@ -32,9 +32,9 @@ in `<ul>` or `<ol>` parent elements:
 
 ## How to fix orphaned list items
 
-Wrap any orphaned `<li>` elements inside a `<ul>` or `<ol>` element.
+Wrap any orphaned `<li>` elements inside a `<ul>`, `<ol>` or `<menu>` element.
```
Comment (heyawhite) [inline src/site/content/en/lighthouse-accessibility/listitem/index.md:35]:
```suggestion
Wrap any orphaned `<li>` elements inside a `<ul>`, `<ol>`, or `<menu>` element.
```

File: src/site/content/en/lighthouse-accessibility/listitem/index.md [hunk 4]
```diff
@@ -32,9 +32,9 @@ in `<ul>` or `<ol>` parent elements:
...
 
 ## Resources
 
-- [Source code for **List items (`<li>`) are not contained within `<ul>` or `<ol>` parent elements** audit](https://github.com/GoogleChrome/lighthouse/blob/master/core/audits/accessibility/listitem.js)
+- [Source code for **List items (`<li>`) are not contained within `<ul>`, `<ol>` or `<menu>` parent elements** audit](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/accessibility/listitem.js)
 - [`<li>` elements must be contained in a `<ul>` or `<ol>` (Deque University)](https://dequeuniversity.com/rules/axe/3.3/listitem)
```
Comment (heyawhite) [inline src/site/content/en/lighthouse-accessibility/listitem/index.md:40]:
```suggestion
- [Lighthouse audit source code](https://github.com/GoogleChrome/lighthouse/blob/master/core/audits/accessibility/listitem.js) for **List items (`<li>`) are not contained within `<ul>`, `<ol>` or `<menu>` parent elements**
- [`<li>` elements must be contained in a `<ul>` or `<ol>`](https://dequeuniversity.com/rules/axe/3.3/listitem), Deque University
```

Comment (heyawhite) [reply src/site/content/en/lighthouse-accessibility/listitem/index.md:40]: Original Lighthouse audit link was broken.

**Review by heyawhite — APPROVED** [review]
LGTM % changes.

**Pull request merged**